src/main/java/io/fabric8/jenkins/openshiftsync/JenkinsUtils.java

```diff
@@ -279,12 +301,19 @@ public static boolean triggerJob(WorkflowJob job, Build build) throws IOExceptio
       }
     }
     
+    ParametersAction userProvidedParams = BuildToParametersActionMap.remove(build.getMetadata().getName());
     // grab envs from actual build in case user overrode default values via `oc start-build -e`
     JenkinsPipelineBuildStrategy strat = build.getSpec().getStrategy().getJenkinsPipelineStrategy();
     // only add new param defs for build envs which are not in build config envs
     addJobParamForBuildEnvs(job, strat, false);
-    // now add the actual param values stemming from openshift build env vars for this specific job
-    buildActions = setJobRunParamsFromEnv(job, strat, buildActions);
+    if (userProvidedParams == null) {
+        LOGGER.fine("setting all job run params since this was started via oc" );
```

Comment:
does this really mean it was started via oc, or could it also mean that job doesn't have any parameters defined on it?